runtime/src/legacy_memory_manager/cpp/Memory.cpp

```diff
@@ -46,6 +46,11 @@
 #include "Runtime.h"
 #include "Utils.h"
 #include "WorkerBoundReference.h"
+#include "Weak.h"
+
+#ifdef KONAN_OBJC_INTEROP
+#include "ObjCExport.h"
```

Comment:
Somewhat innapropriate here.

Reply:
Maybe make a smaller header dedicated to MM-ObjC integration?

Reply:
Yeah, good idea, thanks!